main/install/js/inputShowPwd.js

```diff
@@ -0,0 +1,58 @@
+/*
+*
+* */
+function inputShowPwd(ele) {
+	this.createNewInput = function (oldObject, oType) {
+		var newObject = document.createElement('input');
+		newObject.type = oType;
+		if (oldObject.size) newObject.size = oldObject.size;
+		if (oldObject.value) newObject.value = oldObject.value;
+		if (oldObject.name) newObject.name = oldObject.name;
+		if (oldObject.className) newObject.className = oldObject.className;
+		return newObject;
+	}
+	//初始化样式及效果
+	this.init(ele);
+	return document.getElementsByClassName(ele);
+}
+
+inputShowPwd.prototype.init = function (ele) {
+	var T = this;
+	this.container = $('.' + ele);  //input包装容器
+	this.inputEle = this.container.children('input[type="password"]');   //input输入框
+	this.deleteEle = this.container.children('.showEle'); //显示按钮
+	(function () {
+		//处理ie9不支持栅格键点击触发onpropertychange事件
+		if (navigator.appName == 'Microsoft Internet Explorer') {
+			if (navigator.appVersion.match(/9./i) == '9.') {
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5720016)